src/views/admin/partials/menu.tpl

```diff
@@ -257,6 +257,7 @@
 					</li>
 					{{{end}}}
 					{{{end}}}
```

Comment:
This `{{end}}` is extra?

Reply:
This `{{end}}` does close the `{{{if authentication.length}}}` above, however as there are two conditionals testing the same think [`authentication.length`] this if statement appears to be superfluous/could actually be removed.